pallets/liquidity-pools/src/message.rs

```diff
@@ -201,7 +201,9 @@ impl BatchMessages {
 	TypeInfo,
 	MaxEncodedLen,
 )]
+#[cfg_attr(feature = "runtime-benchmarks", derive(Default))]
 pub enum Message<BatchContent = BatchMessages> {
+	#[cfg_attr(feature = "runtime-benchmarks", default)]
```

Comment:
I think it's ok to add a `default` tag without the burden of `runtime-benchmarks`. Dealing with features always end up with issues in the future. It must be used only if there is a security issue or a high increase of code in the wasm. Any of those is an issue here

Reply:
OK, definitely gonna make our lives easier if we do that. On it.